src/main/resources/i18n/messages.properties

```diff
@@ -440,14 +440,27 @@ user.edit.phone.no-numbers=Phone number must contain numbers
 user.edit.passwordReused=This password has already been used.  Enter a new password.
 user.edit.selectRole=Select Role
 
+
+# ========================================================================================== #
+# User Groups                                                                                #
+# ========================================================================================== #
+UserGroupsPage.title=User Groups
+UserGroupsPage.create=Create New User Group
+UserGroupsTable.name=Name
+UserGroupsTable.description=Description
+UserGroupsTable.created=Created Date
+UserGroupsTable.delete-confirm=Do you really want to delete this user group?  All members of this user group will be removed from the projects that this has been added to.
+UserGroupsTable.delete-tooltip=Delete this user group
+server.usergroups.delete-success=Successfully deleted {0}
+
 # Create and edit groups page
 group.create.details=Create Group
 group.name=Group Name
 group.name.required=Required
 group.description=Group Description
 errors.groupname=Oops, there's something wrong with your group name
 group.name.exists=A group with this name already exists, choose a different name.
-group.id=Group ID
+group.id=Group IDE
```

Comment:
Shouldn't this just be `ID` and not `IDE`?

Reply:
Oops ;) Fixed in 7e29405c458923968efbe9abd8c0638949d9d403